.github/workflows/update-project-on-issue-label.yml

```diff
@@ -43,6 +43,14 @@ jobs:
           cd .github/actions/project_update/
           python project_update.py ${{ github.event.issue.node_id }} --status Blocked
 
+      - name: Update Waiting issues
```

Comment:
@pavish This change seems like a bad merge.

Reply:
@kgodey This was added in https://github.com/centerofci/mathesar/pull/746/commits/12ef48d436e33d47d60c33206b990205e47b38a5, which was mentioned here as harmless: https://github.com/centerofci/mathesar/pull/746#issuecomment-955142173

There wasn't another step with the same name. Should I go ahead and remove this?

Reply:
@pavish it was superseded by https://github.com/centerofci/mathesar/commit/24e615a0c5f618d36e23ca508aa3a8b34036ac2b. I thought I had removed the older one. Please make sure that there's only one and it's the latest one (it should be setting the status to `Waiting` in the last line, not `Blocked`).

Reply:
I think I got confused and forgot that I added it in this PR.

Reply:
Thanks. Updated in https://github.com/centerofci/mathesar/pull/746/commits/e5bfc70150b58c7b5a3854cb58cfab4ecaaac95d